tests/unit/backend/utils/attachment.spec.ts

```diff
@@ -0,0 +1,257 @@
+import { ObjectId } from 'bson'
+import { readFileSync } from 'fs'
+import { cloneDeep, merge } from 'lodash'
+
+import {
+  addAttachmentToResponses,
+  areAttachmentsMoreThan7MB,
+  attachmentsAreValid,
+  handleDuplicatesInAttachments,
+  mapAttachmentsFromParsedResponses,
+} from 'src/app/utils/attachment'
+import {
+  BasicField,
+  IAttachmentResponse,
+  ISingleAnswerResponse,
+} from 'src/types'
+
+const validSingleFile = {
+  filename: 'govtech.jpg',
+  content: readFileSync('./tests/unit/backend/resources/govtech.jpg'),
+  fieldId: String(new ObjectId()),
+}
+
+const invalidSingleFile = {
+  filename: 'invalid.py',
+  content: readFileSync('./tests/unit/backend/resources/invalid.py'),
+  fieldId: String(new ObjectId()),
+}
+
+const zipWithValidAndInvalid = {
+  filename: 'invalidandvalid.zip',
+  content: readFileSync('./tests/unit/backend/resources/invalidandvalid.zip'),
+  fieldId: String(new ObjectId()),
+}
+
+const zipNestedInvalid = {
+  filename: 'nested.zip',
+  content: readFileSync('./tests/unit/backend/resources/nestedInvalid.zip'),
+  fieldId: String(new ObjectId()),
+}
+
+const zipNestedValid = {
+  filename: 'nestedValid.zip',
+  content: readFileSync('./tests/unit/backend/resources/nestedValid.zip'),
+  fieldId: String(new ObjectId()),
+}
+
+const zipOnlyInvalid = {
+  filename: 'onlyinvalid.zip',
+  content: readFileSync('./tests/unit/backend/resources/onlyinvalid.zip'),
+  fieldId: String(new ObjectId()),
+}
+
+const zipOnlyValid = {
+  filename: 'onlyvalid.zip',
+  content: readFileSync('./tests/unit/backend/resources/onlyvalid.zip'),
+  fieldId: String(new ObjectId()),
+}
+
+const MOCK_ANSWER = 'mockAnswer'
+
+describe('attachmentsAreValid', () => {
+  test('returns true for a single valid file', async () => {
+    return expect(attachmentsAreValid([validSingleFile])).resolves.toBe(true)
+  })
+
+  test('returns true for multiple valid files', async () => {
```

Comment:
I realized I have been missing out on the test naming schemes, prefer 
```
it(should .... when ....)
```
format compared to 

```
test(x should y when z)
```

since the former seems to parse easier? Or is it just me?

Plus it conflicts with TS since testcafe has the same global :(. Have you found a fix for that?

Also because `tit` and `tita` is easier to type that `test` and `testa` :P

Reply:
Ahh yes my bad, I changed all the `test` to `it` on my other PR but forgot about this one. Will change it!

Reply:
And no I haven't found a fix for the testcafe global haha